internal/pkg/manifest/workload.go

```diff
@@ -729,6 +729,34 @@ func (p *PlatformArgs) bothSpecified() bool {
 	return (p.OSFamily != nil) && (p.Arch != nil)
 }
 
+// PlatformString returns a specified of the format <os>/<arch>.
+func PlatformString(os, arch string) string {
+	return fmt.Sprintf("%s/%s", os, arch)
+}
+
+// RedirectPlatform returns a platform that's supported for the given manifest type.
+func RedirectPlatform(image, os, arch, wlType string) (platform *string, err error) {
```

Comment:
Can we return a string instead of a pointer? the caller can then decide if they need a pointer
```go
p, _ := RedirectPlatform(os, arch, t)
o.platform = &p
```

Reply:
Yes, sorry-- missed this in the last pass! Done now.